resources/lambda-layers.go

```diff
@@ -0,0 +1,75 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type lambdaLayer struct {
+	svc       *lambda.Lambda
+	layerName *string
+	version   int64
+}
+
+func init() {
+	register("LambdaLayer", ListLambdaLayers)
+}
+
+func ListLambdaLayers(sess *session.Session) ([]Resource, error) {
+	svc := lambda.New(sess)
+
+	layers := make([]*lambda.LayersListItem, 0)
+
+	params := &lambda.ListLayersInput{}
+
+	err := svc.ListLayersPages(params, func(page *lambda.ListLayersOutput, lastPage bool) bool {
+		for _, out := range page.Layers {
+			layers = append(layers, out)
+		}
+		return true
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+
+	for _, layer := range layers {
+
+		var i int64 = int64(1) // is there a better way to do this
+		for i <= *layer.LatestMatchingVersion.Version {
```

Comment:
Shouldn't that be a generic for loop? Something like:

```go
for i := 1; i < *layer.LatestMatchingVersion.Version; i++ {
```

Reply:
Sorry, old habits.  I'll fix and resubmit.